domains/xmswmx.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "mergeconflictlover",
+    "email": "lordhimbarbuana@gmail.com"
+  },
+  "records": {
+    "CNAME": "https://mergeconflictlover.github.io/"
```

CNAME records must not have https:// or any trailing url (/)
```suggestion
    "CNAME": "mergeconflictlover.github.io"
```